app.js

```diff
@@ -0,0 +1,7 @@
+import express from 'express';
+import cors from 'cors';
```

Comment:
import works here, we can change it to require syntax if we would like since we are set up with babel